influxdb3_write/src/persister.rs

```diff
@@ -123,13 +124,20 @@ impl Persister {
     /// a [`LastCacheProvider`].
     ///
     /// This is intended to be used on server start.
-    pub async fn load_last_cache_and_catalog(&self) -> Result<(LastCacheProvider, Catalog)> {
+    pub async fn load_last_cache_and_catalog(
```

Comment:
This can happen in a follow on PR, but I think that the loading of the catalog should be refactored so that it attempts to load from object store and returns an option. If that is None, then the startup process should initialize a new Catalog (with a generated instance id) and then persist that catalog to object storage. Then continue the startup.

Reply:
Sure - I've created an [issue](https://github.com/influxdata/influxdb/issues/25346) for that